manifest/v1alpha/slo/when_delayed.go

```diff
@@ -0,0 +1,11 @@
+package slo
+
+//go:generate ../../../bin/go-enum --nocase --lower --names --values
+
+// WhenDelayed represents enum for behavior of Composite SLO objectives
+/* ENUM(
+CountAsGood
```

Comment:
Unless you want zero value to be CountAsGood:

```suggestion
CountAsGood = 1
```

Reply:
nvm, I didn't notice it's string based

Reply:
As discussed, enum is string based, so this has no effect.